main.md

```diff
@@ -1008,18 +1128,16 @@ In this example, the Issuer is using a following object as a set of claims to is
 }
 ```
 
-The following shows the resulting SD-JWT payload:
+The issuer in this example further adds the two claims `birthdate` and `place_of_birth` to the `claims` element in plain text. The following shows the resulting SD-JWT payload:
 
 {#example-complex-sd_jwt_payload}
 ```json
 {
   "iss": "https://example.com/issuer",
   "cnf": {
-    "jwk" : {
-        "kty": "RSA",
-        "n": "pm4bOHBg-oYhAyPWzR56AWX3rUIXp11_ICDkGgS6W3ZWLts-hzwI3x65659kg4hVo9dbGoCJE3ZGF_eaetE30UhBUEgpGwrDrQiJ9zqprmcFfr3qvvkGjtth8Zgl1eM2bJcOwE7PCBHWTKWYs152R7g6Jg2OVph-a8rq-q79MhKG5QoW_mTz10QT_6H4c7PjWG1fjh8hpWNnbP_pv6d1zSwZfc5fl6yVRL0DV0V3lGHKe2Wqf_eNGjBrBLVklDTk8-stX_MWLcR-EGmXAOv0UBWitS_dXJKJu-vXJyw14nHSGuxTIK2hx1pttMft9CsvqimXKeDTU14qQL1eE7ihcw",
-        "e": "AQAB"
-    }
+    "kty": "RSA",
```

Comment:
why is `cnf` reverting back to not including `jwk` element..?

Reply:
I assume it's because this branch does not yet contain the fix for the `jwk` in `cnf`, but I'm not sure why it was in the example before. I expect that this will fix itself after merging into master.

Reply:
I see that the base branch was merged, that explains this: The base branch now contains the fix, this branch did not yet, so this will be fixed after merging.